src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction.',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction.',
+  },
+});
```

Comment:
- [OPTIONAL] Kindly make sure the initialState for status starts with empty status and changes only when the `checkStatus` action is triggered.

```suggestion
const initialState = {
  categories: [],
  status: '',
};

const categoriesSlice = createSlice({
  name: 'categories',
  initialState,
  reducers: {
    checkStatus: (state) => {
      state.status = "Under Construction"
  },
});
```

Reply:
Thanks for your review and your help @levy002! I will implement the suggested changes.